src/getLocationForJsonPath.ts

```diff
@@ -99,6 +109,38 @@ function findNodeAtPath(node: YAMLNode, path: JsonPath, closest: boolean) {
   return node;
 }
 
+function getMappings(mappings: YAMLMapping[], mergeKeys: boolean): YAMLMapping[] {
+  if (!mergeKeys) return mappings;
+
+  return mappings.reduce<YAMLMapping[]>((mergedMappings, mapping) => {
+    if (isValidNode(mapping as YAMLNode)) {
+      if (mapping.key.value === SpecialMappingKeys.MergeKey) {
+        mergedMappings.push(...reduceMergeKeys((mapping as YAMLMapping).value as YAMLNode));
```

Comment:
Mapping is an element of `mappings` which is a `YAMLMapping` — why is this cast required?

Reply:
What's in `value` usually?

Reply:
![image](https://user-images.githubusercontent.com/9273484/62207612-c3b3a800-b3be-11e9-9f59-84b785148ed5.png)
It's `YAMLNode`, but I get the above. 
Types from dist seem to be different than the types from src. :/

Reply:
Ahhhhh, I'm blind. Wrong import path.